### ISSUE TITLE
Great!

### ISSUE THREAD

**Issue by chelBot** [eyedarts/codecademy-pro-capstone#4]
Included a favicon and put some thought into viewport zooming. As I mentioned in slack, this really is a value judgement but the more I learn about accessibility the more I feel like I should really stop disabling zoom. It's ultimately up to you but in the future I will not be including the maximum-scale=1.0 attribute within my meta tags! Btw, if you are interested in learning a bit more about the meta tag: https://benfrain.com/understanding-the-viewport-meta-tag-and-css-viewport/

https://github.com/eyedarts/codecademy-pro-capstone/blob/master/index.html#L7-L8
